File: cp1_6.20_auto178/validate_cards.py
import json
from collections import Counter

with open(r'e:\solo\SoloAutoDemo\tasks\auto178\backend\card_data.json', 'r', encoding='utf-8') as f:
    data = json.load(f)

print(f'卡牌总数: {len(data)}')
elements = Counter(card['element'] for card in data)
for elem, count in elements.items():
    print(f'{elem}: {count}张')

print('\n卡牌列表:')
for card in data:
    emoji = card['emoji']
    name = card['name']
    element = card['element']
    mana = card['manaCost']
    print(f'  {emoji} {name} ({element}) - 法力:{mana}')

File: cp1_6.20_auto178/validate_cards.js
const fs = require('fs');
const data = JSON.parse(fs.readFileSync('e:\\solo\\SoloAutoDemo\\tasks\\auto178\\backend\\card_data.json', 'utf8'));
console.log('卡牌总数:', data.length);
const elements = {};
data.forEach(card => {
  elements[card.element] = (elements[card.element] || 0) + 1;
});
for (const [elem, count] of Object.entries(elements)) {
  console.log(elem + ':', count + '张');
}
console.log('');
console.log('卡牌列表:');
data.forEach(card => {
  console.log('  ' + card.emoji + ' ' + card.name + ' (' + card.element + ') - 法力:' + card.manaCost);
});
